lc0_main.go

```diff
@@ -46,7 +46,7 @@ var (
 	hasDx           bool
 	testedCudnnFp16 bool
 
-	localHost = "Unknown"
+	localHost = flag.String("localhost", "Unknown", "Localhost name to send to the server when reporting (defaults to Unknown, overridden by settings.json)")
```

Comment:
Make the default here an empty string so that `Unknown` is not a magic word.

Reply:
Done